website/blog/corba.md

```diff
@@ -0,0 +1,184 @@
+# TODO
+
+<div class="blog-author">TODO</div>
+<div class="blog-date">TODO</div>
+
+## Introduction
+
+Service Weaver is a programming framework for writing distributed applications
+in Go. A Service Weaver application is implemented as a set of actor-like
+entities called [components][]. A component implements a set of methods, and
+components interact by calling these methods. When a Service Weaver application
+is deployed, some components are co-located in the same process, and others are
+run in separate processes, potentially on separate machines. As a result, some
+method calls are performed locally, and some are performed remotely.
+
+This paradigm might sound familiar to you. It's similar to the distributed
+object model used by CORBA, DCOM, Java RMI, and countless other frameworks that
+have come and gone. It is [widely believed][why_corba_failed] that these
+frameworks failed because they attempted to make remote method calls **look**
+and **act** like local method calls. This flies in the face of the [fallacies of
```

Comment:
The two sentences starting with `"This flies in the face..."` are making a big cognitive jump from the "look and act"  sentence. i.e., it's not at all obvious what "this" is and why it "flies in the face".

Can you just say that:

`Related research [fallacies] [a_note_on_dc] seemingly supports this theory.`

, or something to that effect?